src/Filament/Resources/AbstractContentResource.php

```diff
@@ -322,26 +325,166 @@ public static function table(Table $table): Table
                     ->sortable(),
             ])
             ->filters([
-                Tables\Filters\TrashedFilter::make(),
-                TernaryFilter::make('is_draft')
-                    ->label('Draft')
-                    ->attribute('is_draft')
-                    ->nullable()
+                SelectFilter::make('status')
```

Comment:
Can we please make this multiple select, where the default is everything _except_ "Deleted" is checked?